cmd/mesh-bootstrap/main.go

```diff
@@ -66,8 +71,35 @@ type Config struct {
 	// P2PAdvertiseAddrs is a comma separated list of libp2p multiaddresses which the
 	// bootstrap node will advertise to peers.
 	P2PAdvertiseAddrs string `envvar:"P2P_ADVERTISE_ADDRS"`
-	// DataDir is the directory used for storing data.
+	// DataStoreType is the data store which will be used to store DHT data
```

Comment:
Looking at these env vars again, I think it is confusing that they aren't separated in any way. Can we prefix the ones related to Postgres/SQL with `POSTGRES` or `SQL` and the ones related to LevelDB with `LEVEL
DB`?

For example `LEVELDB_DATA_DIR` and `SQL_DB_CONNECTION_STRING`.

Reply:
That makes sense, renamed.